wfe-core/src/main/java/ru/runa/wfe/chat/dto/ChatMessageDto.java

```diff
@@ -1,62 +1,33 @@
 package ru.runa.wfe.chat.dto;
 
 import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.ArrayList;
 import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
 import org.apache.commons.lang.StringUtils;
 import ru.runa.wfe.chat.ChatMessage;
 
+@Getter
+@Setter
 public class ChatMessageDto extends ChatDto {
     private ChatMessage message;
+
+    @JsonProperty("fileArray")
     private List<ChatFileDto> filesDto = new ArrayList<ChatFileDto>();
     private boolean old = false;
+
+    @JsonProperty("mentioned")
     private boolean mentionedFlag = false;
```

Comment:
```suggestion
    private boolean isMentioned = false;
```
В java конвенциях обычно так принято. Для `coreUserFlag` и `old` аналогично. Предлагаю заодно поправить